adagrams/game.py

```diff
@@ -1,11 +1,95 @@
+import random
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 
+    'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 
+    'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1, 
+    'Y': 2, 'Z': 1
+    }
+    MAX_HAND_COUNT = 10
+    all_available_let = []
+    current_hand = []
+
+    # creates a list of avaialble 98 letters based on respective frequencis
+    for letter, original_letter_count in LETTER_POOL.items():
+        for letter_index in range(original_letter_count):
+            all_available_let.append(letter)
+
+    # updates initial available list of letters to show remaining letters
+    while len(current_hand) < MAX_HAND_COUNT:
+        available_let_list_index = random.randint(0, len(all_available_let)-1)
+        current_hand.append(all_available_let[available_let_list_index])
+        all_available_let.pop(available_let_list_index)
+
+    return current_hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper() # capitalizes input word
```

Comment:
Nice work on ensuring that all the letters where of the same casing. You could also use `casefold`, which is used for caseless matching. You can find more information on the method [here](https://docs.python.org/3/library/stdtypes.html#str.casefold).

Also, be mindful of re-assigning the parameter of a function. That could cause issues for the caller of the function if the change is not expected. Usually we want to avoid this unless specifically told to.